wagtail/admin/ui/sidebar.py

```diff
@@ -44,14 +44,22 @@ def js_args(self):
 @adapter("wagtail.sidebar.LinkMenuItem", base=BaseSidebarAdapter)
 class LinkMenuItem(MenuItem):
     def __init__(
-        self, name: str, label: str, url: str, icon_name: str = "", classnames: str = ""
+        self,
+        name: str,
+        label: str,
+        url: str,
+        icon_name: str = "",
+        classnames: str = "",
+        attrs: Mapping[str, Any] = None,
```

Comment:
Not entirely sure `Mapping` is correct here – I don’t know how Telepath would serialise a `attrs` that’s not a dict.